### PR TITLE
Add transaction field to Event, deprecate culprits and no longer set …

### PR DESCRIPTION
**Pull request by bretthoerner**
…them automatically in logging integrations.

### CODE REVIEW

**Comment by bretthoerner** [comment]
This should also help the people in #504.

Basically bringing this SDK in line with what Python etc do.

**Comment by bretthoerner** [comment]
This is noisy because of the code structure and tests, but it's a really simple change. 😬 

#java

File: CHANGES [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Version 1.5.7
 -------------
```
Comment (bretthoerner) [inline CHANGES]: I'll actually make this 1.6.0 I think.

File: sentry/src/main/java/io/sentry/event/Event.java [hunk 2]
```diff
@@ -189,6 +193,14 @@ void setCulprit(String culprit) {
         this.culprit = culprit;
```
Comment (tkaemming) [inline sentry/src/main/java/io/sentry/event/Event.java]: Should this also be deprecated?

Comment (bretthoerner) [reply sentry/src/main/java/io/sentry/event/Event.java]: Yep! Done.

**Pull request merged**